tests/browser/ckeditor.jsx

```diff
@@ -278,6 +276,54 @@ describe( 'CKEditor Component', () => {
 				} );
 			} );
 		} );
+
+		it( 'uses the freshest data - component mounted, editor not loaded', () => {
```

Comment:
Please, add tickets references
```
// (#114, #127)
```
Like we have here: https://github.com/ckeditor/ckeditor4-react/blob/04938080004fc385e232da53fdf4e130dda70971/tests/browser/ckeditor.jsx#L69-L70

[Code style reference](https://github.com/ckeditor/ckeditor4/blob/major/dev/docs/codestyle.md#210-comments)

Reply:
Done!